molo/commenting/models.py

```diff
@@ -19,3 +21,9 @@ class MPTTMeta:
     class Meta:
         app_label = 'commenting'
         ordering = ['-submit_date', 'tree_id', 'lft']
+        
+@receiver(comment_was_flagged, sender=MoloComment)
+def remove_comment_if_flag_limit(sender, comment, flag, created, **kwargs):
+    if (comment.flags.count() >= 3):
```

Comment:
@miltontony I have a concern with that. The SoW requires that the threshold be adjustable through the admin interface. Does Django provide admin access to settings such as this or is that something I would have to macgyver?